src/Microsoft.ML.FastTree/TreeEnsembleFeaturizer.cs

```diff
@@ -742,6 +742,8 @@ public static IDataTransform CreateForEntryPoint(IHostEnvironment env, Arguments
                 mapper =
                     (in TInput src, ref Single dst) =>
                     {
+                        // Attention: this method will be used in multipe threading,
+                        // don't put temp variable outside of this method to avoid race condition
```

Comment:
Nitpick: The comment is slightly confusing. Can you please rephrase it? Say, something like:

This method is called from multiple threads. Do not move the temp variable outside this method. If you do, the variable is shared between the threads and results in a race condition.

Reply:
Isnt there a test that can be enabled back because of this fix?

Reply:
No, the test (MulticlassTreeFeaturizedLRTest) is not disabled but I see several fail on machine learning full test set.
As test machine only has 2 cores so the failure rate is not that high but if we run the test locally the failure rate is more than 10 percent.

---
In reply to: [394607002](https://github.com/dotnet/machinelearning/pull/4950#discussion_r394607002) [](ancestors = 394607002)

Reply:
Sure, thanks

---
In reply to: [394606675](https://github.com/dotnet/machinelearning/pull/4950#discussion_r394606675) [](ancestors = 394606675)